tests/hardware/src/forwardBackwardDFMC-MD22.cpp

```diff
@@ -158,8 +158,11 @@ int main(int argc, char* argv[]) {
           CHECK_EXIT_ON_SIGNAL
         }
       }
-      catch(mtca4u::MotorDriverException& e) {
-        std::cout << "Problem reading target position" << std::endl;
+      catch(ChimeraTK::logic_error& e) {
```

Comment:
Why the logic_error may be thrown in this place at all. It looks like only runtime error should be possible here.

Reply:
Ah yes, that is most likely only a runtime error